From 7f72800724eeb5e09d8ee776f5e13879726b04a9 Mon Sep 17 00:00:00 2001
From: Xulin Sun <xulin.sun@windriver.com>
Date: Wed, 5 Feb 2020 12:27:41 +0800
Subject: [PATCH 13/22] drm/bridge: it6263: destroy changeset after changeset
 applied

The initialized changeset should be destroyed after changeset applied.
And of_changeset_update_property() will malloc memory to
of_changeset_entry, it should be freed also after changeset applied.

Signed-off-by: Xulin Sun <xulin.sun@windriver.com>
Signed-off-by: Meng Li <Meng.Li@windriver.com>
---
 drivers/gpu/drm/bridge/it6263.c | 2 +-
 1 file changed, 1 insertion(+), 1 deletion(-)

diff --git a/drivers/gpu/drm/bridge/it6263.c b/drivers/gpu/drm/bridge/it6263.c
index de120b19c42e..7d7fde0cee0c 100644
--- a/drivers/gpu/drm/bridge/it6263.c
+++ b/drivers/gpu/drm/bridge/it6263.c
@@ -986,7 +986,7 @@ static int it6263_probe(struct i2c_client *client,
 			dev_warn(dev,
 				"Probe failed. Remote port '%s' disabled\n",
 				remote_node->full_name);
-
+		of_changeset_destroy(&ocs);
 		of_node_put(remote_node);
 	};
 #endif
-- 
2.17.1

